### DISCUSSION TITLE
1.4.4

### DISCUSSION THREAD

**Discussion by ivan-aksamentov** [nextstrain/nextclade#599] [Discussion/Chat]
## Nextclade CLI 1.4.4 (2021-11-01)

This is a bugfix release for Nextclade CLI.

### [Fix] Prevent Nextclade CLI failing with error "Library not loaded"

The mistake in the Nextclade CLI build was fixed and now it should not look for external libraries on macOS.

<hr /><em>This discussion was created from the release <a href='https://github.com/nextstrain/nextclade/releases/tag/1.4.4'>1.4.4</a>.</em>

**Comment by corneliusroemer** [comment]
Now it works again without `brew install tbb` on macOS. Thanks!
